replica/backup.go

```diff
@@ -270,8 +272,12 @@ func preload(d *diffDisk) error {
 					length++
 				}
 			}
-			d.location[offset] = byte(i)
-			d.UsedBlocks++
+			d.rmLock.Lock()
+			if i != len(d.files) {
```

Comment:
shouldn't this be `d.location[offset] != len(d.files)`? and also, this assumes that new file is created after restart by the time preload starts. We need to handle that assumption.

Reply:
I was ignoring the last file completely, although checking at particular offset is better.